### PR TITLE
Issue forms

### PR DESCRIPTION
**Pull request by habibayassin**
I will add it in ORFS once it is approved here

### CODE REVIEW

**Comment by maliberty** [comment]
Please include a screen shot of what the form look like.

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+        &nbsp;&nbsp;&nbsp;&nbsp;3. Run `make {script}_issue` in the directory "OpenROAD-flow-scripts/flow" where script is wildcarded from the "OpenROAD-flow-scripts/scripts" directory e.g `make cts_issue`
+                                          
+        &nbsp;&nbsp;&nbsp;&nbsp;4. Upload the generated tar file
+
+        B. Upload relevant files
+
+        &nbsp;&nbsp;&nbsp;&nbsp;- Upload a tar file containing the relevant files (.def, .lef and flow.tcl).
+
+        &nbsp;&nbsp;&nbsp;&nbsp;- List the commands used.
```
Comment (vvbandeira) [inline .github/ISSUE_TEMPLATE/bug_report.yml:81]: I think we can have different entries here. One for "Standalone OR", "With OL", "With ORFS",...

**Review by vvbandeira — CHANGES_REQUESTED** [review]

**Comment by habibayassin** [comment]
> Please include a screen shot of what the form look like.

image
image
image
image

**Comment by habibayassin** [comment]
Having 3 separate forms:
image

**Comment by vvbandeira** [comment]
@maliberty please see the prints above from Habiba, also you can try to use [this link](https://github.com/habibayassin/OpenROAD/issues/new/choose) -- works for me at least.

**Comment by maliberty** [comment]
I tried
```
etc/Env.sh 
Please pull the latest changes and try again, if problem persists file a github issue with the re-producible test case.
```
I was in a git repo so this doesn't seem to work correctly.  "git -C . rev-parse" doesn't generate any output and I don't understand why it should.

**Comment by vvbandeira** [comment]
> I was in a git repo so this doesn't seem to work correctly. "git -C . rev-parse" doesn't generate any output and I don't understand why it should.

I don't think this is expected to return anything. The goal of the command is make sure that the folder the user is running on is part of a git repo. In other words, the command will error if the user runs the script outside of the repo -- and will print `git commit: unknown, this is not a git repository.\n(Please make sure that you have the latest code changes and add the commit hash in the description)\n`

@maliberty Is your repo on the master branch and updated with upstream? I get this on my mac:
```
%  etc/Env.sh

Git commit: a48aa12e2931724b8e51a65613f6d317fd302a1e
kernel: Darwin 22.1.0
os:             macOS           13.0.1
cmake version 3.24.2
Apple clang version 14.0.0 (clang-1400.0.29.202)
Apple clang version 14.0.0 (clang-1400.0.29.202)
```

**Comment by maliberty** [comment]
lgtm

**Review by vvbandeira — APPROVED** [review]

**Comment by rovinski** [comment]
This is great - I definitely did not know about this when I made the bug report for ORFS a while back. Hopefully this improves bug report quality!

@vvbandeira @maliberty Maybe it's too aggressive, but it might be worth considering using this bot https://github.com/marketplace/actions/auto-close-issues to automatically close issues which do not follow the format (and automatically reopen them once they are edited to follow the format).

But before that, maybe a next step is to add a form for "feature requests" and/or "questions".

**Comment by maliberty** [comment]
Let's hold off on auto-close and see how it goes.  I agree about feature requests.  GH already provides discussions for questions.

**Pull request merged**